awslogs.conf

```diff
@@ -2,8 +2,15 @@
 state_file = /var/log/awslogs-state
 
 [application-logs]
-file = /var/log/application.log
+file = /var/log/digitalmarketplace/application.log
 log_group_name = ${DM_ENVIRONMENT}-${DM_APP_NAME}
-log_stream_name = {hostname}
-datetime_format = "%Y-%m-%dT%H:%M:%S"
+log_stream_name = Application-{hostname}
```

Comment:
I think we should use `{hostname}` here and send nginx logs to a separate log group (eg `${DM_ENVIRONMENT}-${DM_APP_NAME}-nginx`). The logs have quite different formats, so having a separate log group makes sense in terms of how we're trying to set them up.